python_modules/libraries/dagster-aws/dagster_aws_tests/ecs_tests/launcher_tests/test_environment.py

```diff
@@ -0,0 +1,31 @@
+def test_environment_with_container_context(
```

Comment:
Can you also add a test for setting the environment via dagster.yaml?

Reply:
Something like this: https://github.com/dagster-io/dagster/blob/8455843d675eb8f61028cb1e6e2990a8136f0359/python_modules/libraries/dagster-aws/dagster_aws_tests/ecs_tests/launcher_tests/test_secrets.py#L20-L29

Reply:
ok cool did that in another test in the file